service/terraform-plugin/pom.xml

```diff
@@ -88,6 +88,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.10.1</version>
+        </dependency>
     </dependencies>
```

Comment:
☁️ I was wondering if adding another json processing library is needed. I think we already have jackson library available to ema, can we reuse that library?